homeassistant/components/apple_tv/config_flow.py

```diff
@@ -192,7 +194,7 @@ async def async_step_user(
                     self.device_identifier, raise_on_progress=False
                 )
                 assert self.atv
-                self.context["all_identifiers"] = self.atv.all_identifiers
+                self.all_identifiers = self.atv.all_identifiers
```

Comment:
```suggestion
                self.all_identifiers = set(self.atv.all_identifiers)
```

Reply:
I didn't think this was OK, since the existing code mutates a list provided by `pyatv`.

However, the list is created when calling `all_identifiers`: https://github.com/postlund/pyatv/blob/a613c292c5a7c273459654dc4f375993dfffaa3d/pyatv/interface.py#L1374-L1377

So it should be OK to make a set based on the list, and mutate that?

Reply:
Yes